src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -468,6 +468,16 @@ public static URI getResourceFromClassPath(String filename) throws CheckstyleExc
         return uri;
     }
 
+    /**
+     * Finds a resource with a given name.
+     *
+     * @param name name of the desired resource
+     * @return URI of the resource
+     */
+    public static URL getCheckstyleResource(String name) {
```

Comment:
this method is used only in tests, may be make it package private? to avoid some usages by 3rd party checks

Reply:
done